public/js/controllers.js

```diff
@@ -365,5 +381,55 @@ angular.module('starter.controllers', [])
             alert("All fields are required");
         }
     };
+});
+
```

Comment:
The constructors for the controllers are not chained. I find it more readable this way. It also simplifies minification if we want to automate it (TODO).